pyquil/paulis.py

```diff
@@ -22,17 +22,20 @@
 import numpy as np
 import copy
 
-from typing import Dict, FrozenSet, Iterable, Iterator, List, Optional, Tuple, Union
+from typing import Callable, Dict, FrozenSet, Iterable, Iterator, List, Optional, Sequence, Tuple, Union
+from pyquil.quilatom import ExpressionValueDesignator
```

Comment:
instead of importing this, let's define `PauliCoefficientDesignator = Union[int, float, complex]` and use that in place of `ExpressionValueDesignator`